packaging/container/build.sh

```diff
@@ -10,130 +10,28 @@ set -e
 
 SCRIPT_LOCATION=$(cd "$(dirname "$0")"; pwd)
 
-CVMFS_SOURCE_LOCATION="$1"
-CVMFS_RESULT_LOCATION="$2"
-CVMFS_BUSYBOX="$3"
-CVMFS_TAG="$4"
+CVMFS_RESULT_LOCATION="$1"
+CVMFS_BUSYBOX_URL=${2+"https://ecsft.cern.ch/dist/cvmfs/builddeps/busybox"}
+CVMFS_TAG="$3"
 
-[ ! -z $CVMFS_SOURCE_LOCATION ] || exit 1
 [ ! -z $CVMFS_RESULT_LOCATION ] || exit 1
-[ ! -z $CVMFS_BUSYBOX ] || exit 1
+[ ! -z $CVMFS_BUSYBOX_URL ] || exit 1
 [ ! -z $CVMFS_TAG ] || exit 1
 
-# sanity checks
-# TODO: add externals?
-for d in build rootfs; do
-  if [ -d ${CVMFS_RESULT_LOCATION}/${d} ]; then
-    echo "build directory seems to be used before (${CVMFS_RESULT_LOCATION}/${d} exists)"
-    exit 1
-  fi
-done
-for d in build rootfs; do
-  mkdir -p ${CVMFS_RESULT_LOCATION}/${d}
-done
-
-mkdir ${CVMFS_RESULT_LOCATION}/rootfs/bin
-cp $CVMFS_BUSYBOX ${CVMFS_RESULT_LOCATION}/rootfs/bin/
-${CVMFS_RESULT_LOCATION}/rootfs/bin/busybox --install ${CVMFS_RESULT_LOCATION}/rootfs/bin
-
-cd ${CVMFS_RESULT_LOCATION}/build
-cmake -DBUILD_SERVER=no -DBUILD_RECEIVER=no -DBUILD_GEOAPI=no \
-  -DBUILD_LIBCVMFS=no -DBUILD_LIBCVMFS_CACHE=no \
-  -DINSTALL_BASH_COMPLETION=no \
-  -DEXTERNALS_PREFIX=${CVMFS_RESULT_LOCATION}/externals \
```

Comment:
That's of course a pity that we have to give it up.  Now when we want to debug building the container, we always have to build the full cvmfs including the externals.  Previously, the external builds could stick around and we did not need to rebuild them. Hm, any ideas?